Update reset.min.css (add margin-left to ol lists)

agranadosb opened the pull request:
Description of the issue/feature this PR addresses:
I was working on reports and, when I add an HTML field and i used an ordered list, the list wasn't displayed well:

![imagenMala](https://user-images.githubusercontent.com/34869791/73931914-779b7000-48d9-11ea-926c-7ec21baadafe.png)

Current behavior before PR:
When I added the `margin-left: 40px;` to the file reset.min.css the report was displayed well:

![imagenBuena](https://user-images.githubusercontent.com/34869791/73932018-af0a1c80-48d9-11ea-8d04-f1905153895c.png)


Desired behavior after PR is merged:
So, the ordered lists will be displayed with the hoped behavior



--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

mart-e commented:
Hello,

Thank you for the contribution however could you explain why the `40px`? It is usually a bad idea to hardcode pixels.

Thank you for the CLA signature but it seems there is still an issue.

- you are signing _using_ the address `34869791+agranadosb@users.noreply.github.com`
- you are signing _for_ the address `a...@outlook.com`

The email address in the CLA must be the same as for the signature for the verification to succeed.
You can either change the your [git configuration](https://help.github.com/articles/setting-your-commit-email-address-in-git/) or modify the email in the CLA signature.

Thanks

agranadosb commented:
> Hello,
> 
> Thank you for the contribution however could you explain why the `40px`? It is usually a bad idea to hardcode pixels.
> 
> Thank you for the CLA signature but it seems there is still an issue.
> 
> * you are signing _using_ the address `34869791+agranadosb@users.noreply.github.com`
> * you are signing _for_ the address `a...@outlook.com`
> 
> The email address in the CLA must be the same as for the signature for the verification to succeed.
> You can either change the your [git configuration](https://help.github.com/articles/setting-your-commit-email-address-in-git/) or modify the email in the CLA signature.
> 
> Thanks

Hi,

First of all, thanks for the information about the CLA signature, now I have changed the email settings.

Now, I use 40px because the _ul_ lists use that margin, in line 51 a 40px margin is added to the _ul_ lists, but, in the _ol_ lists there is no margin. 
When I print a report using these two types of lists, the _ul_ was showing well, but the _ol_ list wasn't. I was searching why that was happening, and I found that the problem was in the 40px margin, I added it to the _ol_ lists and the problem disappear.
That's the reason why I have added the 40px margin to the _ol_ lists.

I will repeat the pull request now with the new email.

Thanks and regards!
